docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-requirements.rst

```diff
@@ -33,5 +33,128 @@ The launcher has the following additional requirements on the installation node:
+
+   -  A Slurm partition with GPUs is identified as a CUDA/ROCM resource pool. The type is inherited
+      from the ``resource_manager.slot_type`` configuration. It can be also be specified-per
+      partition using ``resource_manager.partition_overrides``
+
+   -  A Slurm partition with no GPUs is identified as an AUX resource pool.
+
+   -  The Determined default resource pool is set to the Slurm default partition.
+
+-  Tune the Slurm configuration for Determined job preemption.
+
+   Slurm preempts jobs using signals. When a Determined job receives SIGTERM, it begins a checkpoint
+   and graceful shutdown. To prevent unnecessary loss of work, it is recommended to set ``GraceTime
+   (secs)`` high enough to permit the job to complete an entire Determined ``scheduling_unit``.
+
+   To enable GPU job preemption, use ``PreemptMode=REQUEUE`` or ``PreemptMode=REQUEUE``, because
+   ``PreemptMode=SUSPEND`` does not release GPUs so does not allow a higher-priority job to access
+   the allocated GPU resources.
+
+.. _pbs-config-requirements:
+
+******************
+ PBS Requirements
+******************
+
+Determined should function with your existing PBS configuration. The following steps are recommended
+to optimize how Determined interacts with PBS:
+
+-  Configure PBS to manage GPU resources.
+
+   Determined works best when allocating GPUs. By default, Determined selects compute nodes with
+   GPUs using the option ``-select={slots_per_trial}:ngpus=1``. If PBS cannot be configured to
+   identify GPUs in this manner, specify the :ref:`pbs section <cluster-configuration-slurm>`
+   ``gres_supported`` option to ``false`` when configuring Determined, and it will then be the
+   user's responsibility to ensure that GPUs will be available on nodes selected for the job using
+   other configurations such as targeting a specific resource pool with only GPU nodes, or
+   specifying a PBS constraint in the experiment configuration.
+
+-  Ensure homogeneous PBS queues.
+
+   Determined maps PBS queues to Determined resource pools. It is recommended that the nodes within
+   a queue are homogeneous for Determined to effectively schedule GPU jobs.
+
+   -  A PBS queue with GPUs is identified as a CUDA/ROCM resource pool. The type is inherited from
+      the ``resource_manager.slot_type`` configuration. It can be also be specified-per partition
+      using ``resource_manager.partition_overrides``.
+
+   -  A PBS queue with no GPUs is identified as an AUX resource pool.
+
+   -  The Determined default resource pool is set to the PBS default queue.
+
+-  Tune the PBS configuration for Determined job preemption.
+
+   PBS supports a wide variety of criteria to trigger job preemption, and you may use any per your
+   system and job requirements. Once a job is identified for preemption, PBS supports four different
+   options for job preemption which are specified via the preemption_order scheduling parameter. The
```

Comment:
Does "preemption_order" need the two back quotes like you use elsewhere in the document?